packages/sentry/src/sentry.ts

```diff
@@ -17,7 +22,7 @@ export const boot = () => {
   }
 };
 
-export const updateUser = user => {
+export const updateUser = (user: User) => {
   if (user && window.app.trackingSentry) {
```

Comment:
Nitpick: we don't need to check for user anymore
```suggestion
   if (window.app.trackingSentry) {
```

Reply:
Yep, you're right!